.github/workflows/test.yaml

```diff
@@ -8,6 +8,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9.5'
```

Comment:
`actions/setup-python@v2` now defaults to python 3.10, which breaks the build: 
https://github.com/Athena-Home-Loans/keydra/actions/runs/1502156959